fabfile/run.py

```diff
@@ -203,3 +202,32 @@ def tpch_queries(query_info, connectionURI, pg_version, citus_version, config_fi
         out_val = run(run_string)
         results_file.write(out_val)
         results_file.write('\n')
+
+@task
+@roles('master')
+def valgrind(*args): 
+    'Runs valgrind tests'
+
+    # validate arguments
+    if len(args) != 2 or args[0] not in ('enterprise', 'citus'):
+        abort('You must provide two arguments, with a command such as "run.valgrind:citus,v6.0.1".\n\
+            First argument can only be "citus" or "enterprise".')
+
+    citus_repo = args[0]
+    citus_version = args[1]
+
+    # install citus and set citus path variable
+    if citus_repo == 'citus':
```

Comment:
I think we should use the existing logic for deciding whether to download community or enterprise.

Reply:
same for https://github.com/citusdata/test-automation/pull/130#discussion_r357102192

Reply:
done